rules/aip0203/immutable.go

```diff
@@ -0,0 +1,60 @@
+package aip0203
+
+import (
+	"regexp"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+	"google.golang.org/genproto/googleapis/api/annotations"
+)
+
+// This rule inspects the leading comments of each field
+// and if anything looks similar to "Immutable.", it throws
+// a problem.
+//
+// Examples:
+// Incorrect code for this rule:
+//
+//	message Book {
+//	// The title of the book.
+//	// @Immutable
+//	string title = 1;
+//	}
+//
+// or
+//
+//	message Book {
+//	// Immutable. The title of the book.
+//	string title = 1;
+//	}
+//
+//
+// Correct code for this rule:
+//
+//	message Book {
+//	// The title of the book.
+//	string title = 1 [(google.api.field_behavior) = IMMUTABLE];
+//	}
+var immutable = &lint.FieldRule{
+	Name:   lint.NewRuleName("core", "0203", "immutable"),
+	URI:    "http://api.dev/203#guidance",
+	OnlyIf: withoutImmutableFieldBehavior,
+	LintField: func(f *desc.FieldDescriptor) []lint.Problem {
+		return checkLeadingComments(f, immutableRegexp, "IMMUTABLE")
+	},
+}
+
+var immutableRegexp = regexp.MustCompile("(?i).*immutable.*")
+
+func withoutImmutableFieldBehavior(f *desc.FieldDescriptor) bool {
+	opts := f.GetFieldOptions()
```

Comment:
Can you use [`GetFieldBehavior`](https://github.com/googleapis/api-linter/blob/2f7e4b96ffe9dbebfbef2e04d360dfe2eacf4783/rules/internal/utils/extension.go#L25) in the `rules/internal/utils` package?